purchase_request_procurement/__manifest__.py

```diff
@@ -4,7 +4,7 @@
 
 {
     'name': "Purchase Request Procurement",
-    'version': "10.0.1.0.1",
+    'version': "10.0.1.2.1",
```

Comment:
10.0.1.1.0

same for folder of migration scripts 😅

Reply:
It is already 10.0.1.2.1 :stuck_out_tongue:

Reply:
What do you mean? https://github.com/OCA/purchase-workflow/blob/10.0/purchase_request_procurement/__manifest__.py#L7

Reply:
I mean, the migration folder is 10.0.1.2.1, same as manifest.